time stamp
uint8 channel
uint16 measurement

